_pages/welcome-to-TTS/classes/intro-to-TTS-Tech-Portfolio.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: Intro to 18F Infrastructure
+title: Intro to TTS Tech Portfolio
 navtitle: Infrastructure
 outdated: true
```

Comment:
Another too-late question: is this still outdated after all these updates?

Reply:
Yes, this material is still outdated: We had a card to revamp the onboarding materials, but not sure what happened to it.